examples/components/http-hello-world/README.md

```diff
@@ -0,0 +1,184 @@
+# WASI `http-hello-world` in JavaScript
+
+This folder contains a WebAssembly Javascript component that uses [`wasi:http`][wasi-http] for enabling HTTP handlers in Javascript.
+
+It uses [`jco`][jco] to:
+
+- Generate a WebAssembly component (via `jco componentize`) that can be executed by a WebAssembly runtime (ex. [`wasmtime serve`][wasmtime])
+
+[nodejs]: https://nodejs.org
+[jco]: https://bytecodealliance.github.io/jco/
+[wasi-http]: https://github.com/WebAssembly/wasi-http
+[wasmtime]: https://github.com/bytecodealliance/wasmtime
+
+# Quickstart
+
+## Dependencies
+
+First, install required dependencies:
+
+```console
+npm install
+```
+
+> [!NOTE]
+> As this is a regular NodeJS project, you can use your package manager of choice (e.g. `yarn`, `pnpm`)
+
+At this point, since this project is *just* NodeJS, you could use the module from any NodeJS project or browser project where appropriate.
+
+That said, we'll be focusing on building the JS code we've written so far into a WebAssembly binary, which can run *anywhere* WebAssembly runtimes are supported,
+including in other languages, and the browser.
+
+## Building the WebAssembly component
+
+We can build a WebAssembly component binary out of this JS project with `jco`:
+
+```console
+npm run build
+```
+
+A WebAssembly binary will be written to `string-reverse.wasm`.
+
+## Serving web requests with the WebAssembly component
+
+To run the component and serve requests we can either use `jco` or `wasmtime`:
+
+```console
+$ jco serve http-hello-world.wasm
+Server listening on 8000...
+```
```

Comment:
I think we should rather not recommend `jco serve` or alternatively be very clear that this is for development only.

Reply:
Ah yeah -- I'll add a note that this is for development only!